BAZProject/MovieDetail/MovieDetailViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MovieDetailViewController.swift
+//  BAZProject
+//
+//  Created 1030364 on 17/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MovieDetailViewController: UIViewController, MovieDetailViewProtocol {
+
+	var presenter: MovieDetailPresenterProtocol?
+    var movie: MovieModel?
+
+    @IBOutlet weak var image: UIImageView!
+    @IBOutlet weak var lblTitle: UILabel!
+    @IBOutlet weak var lblReleaseDate: UILabel!
+    @IBOutlet weak var lblVotes: UILabel!
+    @IBOutlet weak var lblOverview: UITextView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configView()
+    }
+
+    func configView() {
+        if let movie = movie {
+            lblTitle.text = movie.movie.title
+            lblReleaseDate.text = "Fecha de estreno: \(movie.movie.releaseDate ?? "NA")"
+            lblVotes.text = "Votos: \(String(movie.movie.voteAverage ?? 0.0))"
+            lblOverview.text = movie.movie.overview
+            image.layer.cornerRadius = 10.0
+            image.image = movie.movieBackdrop
+        }
+    }
```

Comment:
De ser posible, eliminémos este if/else enviando movie como parámetro del método:
```suggestion
    func configView(movie: MovieModel) {
         lblTitle.text = movie.movie.title
         lblReleaseDate.text = "Fecha de estreno: \(movie.movie.releaseDate ?? "NA")"
         lblVotes.text = "Votos: \(String(movie.movie.voteAverage ?? 0.0))"
         lblOverview.text = movie.movie.overview
         image.layer.cornerRadius = 10.0
         image.image = movie.movieBackdrop
    }
```
Si no es posible entonces reemplacemos el `if` con un `guard`:

```suggestion
    func configView() {
        guard let movie = movie  else { return }
        lblTitle.text = movie.movie.title
        lblReleaseDate.text = "Fecha de estreno: \(movie.movie.releaseDate ?? "NA")"
        lblVotes.text = "Votos: \(String(movie.movie.voteAverage ?? 0.0))"
        lblOverview.text = movie.movie.overview
        image.layer.cornerRadius = 10.0
        image.image = movie.movieBackdrop
    }
```